### ISSUE TITLE
In RStudio 1.3, RStudio dictionaries are not detected

### ISSUE THREAD

**Issue by GegznaV** [ropensci/hunspell#42]
In RStudio 1.3, the directory where downloaded RStudio dictionaries are stored has changed. So `hunspell:::rstudio_dicpaths()` finds only 4 default dictionaries. Previously, it found all dictionaries installed by RStudio. Could **hunspell** be updated for RStudio 1.3?

***

**Details**

On Windows, the new directory with dictionaries is:
```r
fs::path(Sys.getenv("APPDATA"), "RStudio/dictionaries/languages-system")
```
e.g.:
```
C:/Users/User/AppData/Roaming/RStudio/dictionaries/languages-system
```
On the other OS'es, it should be (I cannot check this. More details in [this post](https://blog.rstudio.com/2020/02/18/rstudio-1-3-preview-configuration/)):
```r
fs::path_expand_r("~/.config/rstudio/dictionaries/languages-system")
```

In RStudio 1.2 or earlier, it was:
```r
fs::path(rstudioapi::userDictionariesPath(), "languages-system")
```
e.g.:
```
C:/Users/User/AppData/Local/RStudio-Desktop/dictionaries/languages-system
```

**Comment by jeroen** [comment]
Can you suggest a PR to conditionally fix the path based on the version of rstudio?

**Comment by GegznaV** [comment]
Now I think that it is not a good idea to condition the output based on the version of RStudio as this would not allow using the package outside RStudio.

**Comment by GegznaV** [comment]
Starting with RStudio 1.3.x, RStudio contains additional dictionaries in the configuration directory (see [1](https://blog.rstudio.com/2020/02/18/rstudio-1-3-preview-configuration/#the-configuration-folder) or [2](https://docs.rstudio.com/ide/desktop-pro/latest/settings.html#other-settings)), which path could be changed by the environment variables `XDG_CONFIG_DIRS`, `XDG_CONFIG_HOME`, `RSTUDIO_CONFIG_DIR` and `RSTUDIO_CONFIG_HOME` (see [3](https://docs.rstudio.com/ide/desktop-pro/latest/settings.html) or [4](https://blog.rstudio.com/2020/02/18/rstudio-1-3-preview-configuration/#administration-and-the-xdg-standard)). These changes are addressed in PR #48.
